classes/File.js

```diff
@@ -127,7 +135,7 @@ class File {
 
       let params = {
         "message": `Delete file: ${fileName}`,
-        "branch": "staging",
+        "branch":`${this.branchRef}`,
```

Comment:
Refer to above

Reply:
Fixed in [fc2f3](https://github.com/isomerpages/isomercms-backend/commit/fc2f3cb1fc67fd143ef4ae2f0feffd6ff5953472)